### ISSUE TITLE
Add meteor support

### ISSUE THREAD

**Issue by dcworldwide** [google/material-design-lite#1040]
Please adopt https://github.com/MeteorPackaging/autopublish.meteor.com to auto-package/release MDL for Meteor apps. More info here https://autopublish.meteor.com/

**Comment by Garbee** [comment]
I don't think we directly support Meteor yet (since out componentHandler isn't bound to window by default but instead is defined by a var statement in the global area.) Plus handling direct support like this would open up other projects to want direct support which can become time-consuming to maintain support in core with testing to make sure nothing breaks across each platform.

**Comment by sergiotapia** [comment]
Would love baked in Meteor support this! Materialize is great in this regard.

**Comment by SeanBannister** [comment]
Just wanted to add my interest in this as we use Meteor. I've noticed a number of large projects starting to support Meteor \*cough* bootstrap \*cough*. I totally understand the concerns about maintaining it, I do believe once the package.js file is in place it should require minimal maintenance. Maybe someone more familiar with this packaging process could comment.

**Comment by chip** [comment]
@splendido and I have worked on 2 packages for [jsPDF](https://atmospherejs.com/jspdf) and made them the official wrappers on Atmosphere for the jsPDF namespace using the autopublish technique. I'd be interested in assisting with this effort for this package if the maintainer is interested.

[Please see the jspdf-core-wrapper repo](https://github.com/MeteorPackaging/jspdf-core-wrapper/) for an example of the files required. Basically, it should entail us adding the following files: `autopublish.json`, `meteor-pre.js`, `meteor-post.js` and `package.js`. There might also need to be a few minor tweaks to `gulpfile.js` and `package.json`, although I'd need to dig further to be sure.

Once these changes are made, future updates should be seamlessly deployed to Atmosphere (provided that you setup the Github hook per https://autopublish.meteor.com/howitworks, which should only take a minute). 

Please let me know your thoughts. Thanks.

**Comment by sindresorhus** [comment]
:-1: This is a slippery slope and better handled externally by the Meteor community.

**Comment by chip** [comment]
I agree with @sindresorhus and need to clarify my previous comment. 

The files I mentioned will be added to a **separate repo** that the [Meteor Packaging team](https://github.com/MeteorPackaging/) will manage.

The only thing that will need to be done for this repo is that the owner will need to setup the Github hook so that autopublish.meteor.com is notified when commits are merged into master. I will touch base with @splendido to see what needs to be done to expedite this effort.

Sorry for the confusion.

**Comment by sindresorhus** [comment]
Why doesn't Meteor just subscribe to the GitHub releases feed? That way you don't have to bother every repo you want a Meteor variant of to set up a commit hook.

**Comment by chip** [comment]
That is a great idea!  I will look into this further with @splendido since he is the author of the autopublish site. Thank you for the suggestion.

**Comment by Garbee** [comment]
Even if it is just this repo setting up a hook, I believe that is too much for us to do for something we aren't directly supporting. Package publishers shouldn't need the repos to do anything for getting updates. That implies some level of support from us which we are incapable of providing.

**Comment by surma** [comment]
@Garbee If we support one, we’d have to support all. As @sindresorhus said, slippery slope.

We don’t have the manpower to take full responsibility for this. I hope you can get a hook set up on Meteor’s side :)
